contrib/cirrus/setup.sh

```diff
@@ -10,6 +10,11 @@ source $(dirname $0)/lib.sh
 
 req_env_var OS_RELEASE_ID OS_RELEASE_VER GOSRC IN_PODMAN_IMAGE
 
+#####
+##### FIXME. /etc/containers/storage.conf should have a driver name set
+##### Remove when VMs updated
+sed 's/^driver.*=.*""/driver = "overlay"/g' -i /etc/containers/storage.conf
```

Comment:
Why are there two double quotes here? (in dot star doublequote doublequote slash)

Also, I don't know know what this does or why; is there an older version of storage.conf that has the wrong value? Or is `driver` commented out? If commented out, do you need a `#` between the caret and driver in the LHS?

Sorry for asking stupid questions, I really don't understand what's going on because on my laptop storage.conf looks fine

Reply:
It is just looking to see if the driver is set,  By default there is a storage.conf with

driver = ""

On these machines.
So I am changing them to 
driver = "overlay"

If it are set, then we do nothing.

The default storage.conf does not have this set, it is set by the distributions though.  I don't know why these storage.conf files are showing up. Unless they are checked directly out of containers/storage.

Reply:
In older versions of container/storage we ignored these not being set and just defaulted to overlay, but this caused us issues where other values in the storage.conf were being ignored.

driver = ""
overlay.mount_program=fuse-overlayfs

Would not set the mount_porgram

While
driver = "overlay"
overlay.mount_program=fuse-overlayfs

Would.

So rather then assume what the user meant when he did not set this flag, we now print a large warning.

Reply:
>The default storage.conf does not have this set

Thank you for the explanation. Could it be this? If so your sed is not going to do anything because there's no driver line at all.
https://github.com/containers/buildah/blob/d69f76ab1741c834303b9dfc8d03813fe3a59577/contrib/cirrus/lib.sh#L181-L194

Reply:
Excellent, that is it.